index/start-page.njk

```diff
@@ -60,7 +60,7 @@
            <li>in England and Wales <a href="https://www.victimandwitnessinformation.org.uk/">visit the Victim and Witness Information website</a></li>
            <li>in Scotland <a href="https://www.mygov.scot/victim-witness-support/">visit the mygov.scot website</a></li>
         </ul>
-        <p class="govuk-body">You do not need a legal representative to make an application.</p>
+        <p class="govuk-body ga-event ga-event--scrollto" ga-event-action="scrollTo" ga-event-label="bottom" ga-event-category="start-page">You do not need a legal representative to make an application.</p>
```

Comment:
Do these align to? https://support.google.com/analytics/answer/1033068#Anatomy

Some of these could be set with non-custom attributes e.g. using
- The scroll handler
- window.location or something? 

The custom attributes should be prefixed with "data-"